docs/graphql/manual/deployment/downgrading.rst

```diff
@@ -83,1218 +83,17 @@ Hasura GraphQL engine versions - Catalogue version mapping
 Downgrading across catalogue versions
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-You can downgrade the catalogue from a particular version to its previous version by executing the appropriate SQL statements provided below.
+You can downgrade the catalogue from a particular version to a previous version by executing the ``graphql-engine`` executable on the command line,
+with the ``downgrade`` command, specifying the desired catalog version (from the table above) using the ``--to`` option.
 
```

Comment:
Do you think we can add an example of how this CLI command will look?

Reply:
I think we should be very explicit that this is *not* a command that is part of the Hasura CLI but rather a command on `graphql-engine` itself. Therefore, we should be very clear about the following points:

  1. The way to execute this command is to run `docker run hasura/graphql-engine:<VERSION> graphql-engine downgrade ...`.

  2. You need to use a *newer* version of `graphql-engine` to downgrade to an *older* version, since only the newer version knows how to downgrade from that point in time. After you’ve executed the `downgrade` command using the newer version, *then* you should switch to the older version and run `graphql-engine serve` as normal.

Eventually it would probably make sense to add a command to the Hasura CLI that abstracts away some of this stuff, but I think the command on `graphql-engine` is still very useful on its own.